### PR TITLE
fix(apiKey): expose ApiKeyACL type

### PR DESCRIPTION
**Pull request by bodinsamuel**
- Add ApiKeyACL type so we can strictly type the ACL's of api keys

The list can be found here:
https://www.algolia.com/doc/guides/security/api-keys/#rights-acl

### CODE REVIEW

**Comment by bodinsamuel** [comment]
@Ant-hem @aseure nuno asked me to ping you to see if you were interested to have this type in your client's language ☺️

File: packages/client-search/src/types/ApiKey.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export type ApiKeyACL =
```
Comment (nunomaduro) [inline packages/client-search/src/types/ApiKey.ts:1]: You can you create a file named `ApiKeyAclType` with contents similar too: https://github.com/algolia/algoliasearch-client-javascript/blob/master/packages/client-search/src/types/BatchActionType.ts?

Comment (nunomaduro) [inline packages/client-search/src/types/ApiKey.ts:1]: You can you create a file named `ApiKeyAclType` with contents similar too: https://github.com/algolia/algoliasearch-client-javascript/blob/master/packages/client-search/src/types/BatchActionType.ts?

**Comment by nunomaduro** [comment]
All tests passing.

**Pull request merged**